### DISCUSSION TITLE
[Feature request] Deploy to environment  - next minor/major version

### DISCUSSION THREAD

**Discussion by PeterDerks** [microsoft/AL-Go#1579] [Ideas]
Hi Freddy,

I want to request a new feature in the action "Publish to Environment". Can you build a feature that you can choose to deploy the app in  "Current version", "next minor version", "next major version"?
Just like when you manually upload/install an app in BC?

<img width="469" alt="image" src="https://github.com/microsoft/AL-Go/assets/33145826/c37a456a-3ba4-4cda-832c-bffc18a2c745">

This feature will be very handy when the customer tests something in a newer version than the production environment. 

Thanks a lot!

**Comment by freddydk** [comment]
Sounds like a good idea
